assets/src/components/detours/diversionPage.tsx

```diff
@@ -137,7 +143,7 @@ export const DiversionPage = ({
           )}
           {routingError?.type === "unknown" && <RoutingErrorAlert />}
           <DetourMap
-            originalShape={originalRoute.shape.points}
+            originalShape={shape?.points ?? []}
```

Comment:
note: is this a breaking change? Not sure what to do here since we can't derive shapes from the route pattern and expect that they're always there.